tests/parameter_search/test_p_error_binary_search.py

```diff
@@ -371,21 +366,33 @@ def test_binary_search_for_built_in_models(model_class, parameters, threshold, p
 
 
 @pytest.mark.parametrize("is_qat", [False, True])
-def test_invalid_estimator_for_custom_models(is_qat):
+def test_invalid_estimator_for_custom_models(is_qat, load_data):
     """Check that binary search raises an exception for unsupported models."""
 
+    model_name = "CustomModel"
     model = keras.Sequential(
         [
-            keras.layers.Reshape(target_shape=(28 * 28,), input_shape=(28, 28)),
-            keras.layers.Dense(units=256, activation="relu"),
-            keras.layers.Dense(units=192, activation="relu"),
-            keras.layers.Dense(units=128, activation="relu"),
-            keras.layers.Dense(units=10, activation="softmax"),
+            keras.layers.InputLayer(
+                input_shape=(MODELS_ARGS[model_name]["dataset"]["n_features"],)
+            ),
+            keras.layers.Dense(units=32, activation="relu"),
+            keras.layers.Dense(units=16, activation="relu"),
+            keras.layers.Dense(
+                units=MODELS_ARGS[model_name]["dataset"]["n_classes"], activation="softmax"
+            ),
         ]
     )
 
-    dataset = get_torchvision_dataset(DATASETS_ARGS["FashionMNIST"], train_set=True)
-    x_calib, y = data_calibration_processing(dataset, n_sample=1)
+    # dataset = get_torchvision_dataset(DATASETS_ARGS["FashionMNIST"], train_set=True)
+    # x_calib, y = data_calibration_processing(dataset, n_sample=1)
```

Comment:
did you leave the comments in purpose or is it an oversight?

Reply:
An oversight, thanks for seeing it!